service/resource/s3workercloudconfigv1/create.go

```diff
@@ -0,0 +1,48 @@
+package s3workercloudconfigv1
+
+import (
+	"context"
+	"strings"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/service/s3"
+	"github.com/giantswarm/microerror"
+)
+
+func (r *Resource) ApplyCreateChange(ctx context.Context, obj, createChange interface{}) error {
+	s3PutInput, err := toPutObjectInput(createChange)
+	if err != nil {
+		return microerror.Mask(err)
+	}
+
+	_, err = r.awsClients.S3.PutObject(&s3PutInput)
```

Comment:
I think we should do the standard logging with creating the resource and whether it was created or has already been created.

https://github.com/giantswarm/aws-operator/blob/87c7d592b80c256dafb1ddc204c472989d2dcce3/service/resource/namespacev1/create.go#L18

Reply:
Good point, done